csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -4007,7 +4007,7 @@ Example which fails the test:
 
 ### 4.3.7 Use of self referencing URLs Failing to Resolve
 
-For each item in an array of type `references_t` with the category `self` it must be tested that the URL referenced resolves with a HTTP status code less than 400. 
+For each item in an array of type `references_t` with the category `self` it must be tested that the URL referenced resolves with a HTTP status code less than 400.
```

Comment:
Hm, I would trade the elegance of the less than operator for the clarity of excluding the informative HTTP response codes of the 100 family. I think we want to say that the URL is in fact dereference-able. Maybe a `Continue`, `Processing`, or similar is not helpful.

Reply:
Possibly resolved in f67d3b4.